third_party/uncrustify.cfg

```diff
@@ -702,14 +702,14 @@ indent_paren_open_brace         = false    # false/true
 indent_cs_delegate_brace        = false    # false/true
 
 # Whether the 'namespace' body is indented.
-indent_namespace                = false    # false/true
+indent_namespace                = true    # false/true
```

Comment:
I think this should NOT be true... Most large projects do NOT indent namespaces... (4ebad1)